examples/contour/03-envoy.yaml

```diff
@@ -117,6 +122,8 @@ spec:
         - name: envoycert
           mountPath: /certs
           readOnly: true
+        - name: envoy-admin
```

Comment:
does the bootstrap container need this mounted? I think the bootstrap command just generates the config with the requested address and doesnt talk on the socket to envoy since it runs before the envoy container exists

Reply:
nope, that's a mistake.